src/datastar_py/attributes.py

```diff
@@ -113,7 +113,7 @@
 
 class AttributeGenerator:
     def __init__(self, alias: str = "data-") -> None:
-        """A helper which can generate all the Datastar attributes.
+        """Generate all the Datastar attributes.
```

Comment:
This is to make the 'imperative mood' rule pass? I do like that rule in general, but sometimes it a bit annoying, especially when documenting a class or decorator. Perhaps we should disable it.